src/components/core/geoChart/tests/GeoChartLegendRenderer.spec.tsx

```diff
@@ -0,0 +1,156 @@
+                                    },
+                                },
+                            ],
+                        },
+                    },
+                ],
+            },
+            {
+                headers: [
+                    {
+                        attributeHeader: {
+                            identifier: "30.df",
+                            uri: "/gdc/md/storybook/obj/30.df",
+                            name: "City",
+                            localIdentifier: "location",
+                            formOf: {
+                                uri: "/gdc/md/storybook/obj/30",
+                                identifier: "30",
+                                name: "City",
+                            },
+                        },
+                    },
+                ],
+            },
+        ],
+    };
+    return {
+        executionResponse,
+        executionResult,
+    };
+}
+function getGeoConfig(): IGeoConfig {
+    const config: IGeoConfig = {
+        mdObject: {
+            visualizationClass: {
+                uri: "local:pushpin",
+            },
+            buckets: [
+                {
+                    localIdentifier: "size",
+                    items: [
+                        {
+                            measure: {
+                                localIdentifier: "size",
+                                definition: {
+                                    measureDefinition: {
+                                        item: {
+                                            uri: "/gdc/md/storybook/obj/21",
+                                        },
+                                    },
+                                },
+                            },
+                        },
+                    ],
+                },
+                {
+                    localIdentifier: "location",
+                    items: [
+                        {
+                            visualizationAttribute: {
+                                localIdentifier: "location",
+                                displayForm: {
+                                    uri: "",
+                                },
+                            },
+                        },
+                    ],
+                },
+            ],
+        },
+    };
+    return config;
+}
+function createComponent(customProps: IGeoChartLegendRendererProps) {
+    const legendProps = {
+        ...customProps,
+    };
+    const Wrapped = withIntl(GeoChartLegendRenderer);
+    return mount(<Wrapped {...legendProps} />);
+}
+
+describe.only("GeoChartLegendRenderer", () => {
+    const props = {
+        execution: getExecutionResponse(),
+        config: getGeoConfig(),
+        locale: "en-US",
+    };
+    it("should render component with PushpinSizeLegend", () => {
+        const wrapper = createComponent(props);
+        expect(wrapper.find(".geo-legend")).toHaveLength(1);
```

Comment:
Please add a new class-name `s-geo-legend`.
We use the classnames with `s-` prefix for testing :)